package.json

```diff
@@ -12,7 +12,7 @@
   "devDependencies": {
     "autoprefixer": "^8.6.5",
     "graceful-fs": "*",
-    "gulp": "*",
+    "gulp": "3*",
```

Comment:
The config doesn't work with gulp v4, which will be installed by default when set to `*`